CONTRIBUTING.md

```diff
@@ -58,12 +58,11 @@ For each pull request, you are expected to test the defaults to make sure no reg
 - Download and install the current NodeJS LTS version.
 - Run `npm install` in this folder. This installs all necessary npm modules in both the client and server folder.
 - Open VS Code on this folder.
-- Press Ctrl+Shift+B to compile the client and server.
-- Switch to the Debug viewlet.
-- Select `Launch Client` from the drop down.
-- Run the launch config.
-- If you want to debug the server as well use the launch configuration `Attach to Server`.
-- In the [Extension Development Host] instance of VSCode, open a SAS file.
+- Switch to the `Run and Debug` view in the VS Code Activity Bar (Ctrl+Shift+D).
+- Select `Launch Client` from the drop down (if it is not already).
+- Press ▷ to run the launch config (F5).
+- In the [Extension Development Host] instance of VS Code, open a SAS file.
+- _Optional_: If you want to debug the language server as well, also run the launch configuration `Attach to Server`.
```

Comment:
In this instance, do I need to run both `Launch Client` _and_ `Attach to Server`?

Reply:
Yes. So I make it an additional optional step. "Attach" has to be **after** "Launch".